testing
2015-08-21 13:19:25.134
ruby -v -W0 /home/greg/Desktop/src/Open-Table-Explorer/test/data_sources/repository/single_test_fail.rb
ruby 1.9.3p194 (2012-04-20 revision 35410) [x86_64-linux]
Run options: --seed 21693

# Running:

E.

Finished in 0.003230s, 619.1550 runs/s, 0.0000 assertions/s.

  1) Error:
SingleFailTest#test_fail:
RuntimeError: 
    /home/greg/Desktop/src/Open-Table-Explorer/test/data_sources/repository/single_test_fail.rb:13:in `test_fail'
    /usr/lib/ruby/vendor_ruby/minitest/test.rb:108:in `block (3 levels) in run'
    /usr/lib/ruby/vendor_ruby/minitest/test.rb:205:in `capture_exceptions'
    /usr/lib/ruby/vendor_ruby/minitest/test.rb:105:in `block (2 levels) in run'
    /usr/lib/ruby/vendor_ruby/minitest/test.rb:256:in `time_it'
    /usr/lib/ruby/vendor_ruby/minitest/test.rb:104:in `block in run'
    /usr/lib/ruby/vendor_ruby/minitest.rb:332:in `on_signal'
    /usr/lib/ruby/vendor_ruby/minitest/test.rb:276:in `with_info_handler'
    /usr/lib/ruby/vendor_ruby/minitest/test.rb:103:in `run'
    /usr/lib/ruby/vendor_ruby/minitest.rb:779:in `run_one_method'
    /usr/lib/ruby/vendor_ruby/minitest.rb:306:in `run_one_method'
    /usr/lib/ruby/vendor_ruby/minitest.rb:294:in `block (2 levels) in run'
    /usr/lib/ruby/vendor_ruby/minitest.rb:293:in `each'
    /usr/lib/ruby/vendor_ruby/minitest.rb:293:in `block in run'
    /usr/lib/ruby/vendor_ruby/minitest.rb:332:in `on_signal'
    /usr/lib/ruby/vendor_ruby/minitest.rb:319:in `with_info_handler'
    /usr/lib/ruby/vendor_ruby/minitest.rb:292:in `run'
    /usr/lib/ruby/vendor_ruby/minitest.rb:155:in `block in __run'
    /usr/lib/ruby/vendor_ruby/minitest.rb:155:in `map'
    /usr/lib/ruby/vendor_ruby/minitest.rb:155:in `__run'
    /usr/lib/ruby/vendor_ruby/minitest.rb:129:in `run'
    /usr/lib/ruby/vendor_ruby/minitest.rb:56:in `block in autorun'

2 runs, 0 assertions, 0 failures, 1 errors, 0 skips

